source/index.txt

```diff
@@ -7,18 +7,16 @@ Cluster-to-Cluster Sync
 .. default-domain:: mongodb
 
 {+c2c-product-name+} provides continuous data synchronization or a 
-one-time data migration between MongoDB clusters. You can enable
-{+c2c-product-name+} with the :ref:`mongosync <c2c-mongosync>` utility.
+one-time data migration between MongoDB clusters. See the :ref:`<mongosync-considerations>`
```

Comment:
[suggestion] We typically follow the sentence structure of "to do x, see y," so I'd recommend swapping around the clauses on this second sentence: 

```suggestion
one-time data migration between MongoDB clusters. For current considerations on ``mongosync`` usage, see the :ref:`<mongosync-considerations>`
```